### ISSUE TITLE
build failed for Mac 

### ISSUE THREAD

**Issue by endless-horizon** [xournalpp/xournalpp#1387]
```
[ 98%] Built target xournalpp-core
[ 98%] Linking CXX executable xournalpp
ld: framework not found -lCocoa
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
how can I install cocoa.framework for Mac
there, I can't found this app anywhere?

**Comment by andreasb242** [comment]
We still have some issues with the Mac Build.

Generally, you need to have installed XCode, this contains also all needed OS X Frameworks.

But additionally, there are some issues. The common one (because it might be the same for you):
in the Linker command we had:
`-framework -lCoreAudio`
This is wrong.

It should be
`-framework CoreAudio`

Can you check, if you have the same with Cocoa? (maybe call with -v)

**Comment by LittleHuba** [comment]
Fixed by #1353
